plugins/module_utils/turbo/module.py

```diff
@@ -1,6 +1,7 @@
 import json
 import os
 import socket
+import subprocess
```

Comment:
I defer the loading of `subprocess` on purpose. We only need it if the service is not running yet.